### PR TITLE
Embed Privacy Sandbox videos

### PR DESCRIPTION
**Pull request by samdutton**
I've used iframe code for the playlist.

### CODE REVIEW

**Comment by heyawhite** [comment]
@samdutton is this out of date now? Or should I approve and merge?

**Comment by samdutton** [comment]
> @samdutton is this out of date now? Or should I approve and merge?

I'd totally forgotten about this! 

Yes please to approve and merge.

**Review by heyawhite — APPROVED** [review]

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ subhead: The Privacy Sandbox is a series of proposals to satisfy third-party use
 authors:
   - samdutton
 date: 2020-04-08
-updated: 2021-09-20
```
Comment (Hatch11251) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: 7

File: src/site/content/en/blog/trust-tokens/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ subhead: Trust Tokens is a new API to enable a website to convey a limited amoun
 authors:
   - samdutton
 date: 2020-06-22
-updated: 2021-10-28
```
Comment (Hatch11251) [inline src/site/content/en/blog/trust-tokens/index.md:7]: 12/9/21???

File: src/site/content/en/blog/trust-tokens/index.md [hunk 3]
```diff
@@ -31,7 +31,9 @@ Find out more: [What's TrustTokenV3?](https://bit.ly/what-is-trusttokenv3).
 
 <br><br>
 
-{% YouTube 'WnCKlNE52tc' %}
```
Comment (Hatch11251) [inline src/site/content/en/blog/trust-tokens/index.md:34]: vid of what?

**Pull request merged**